src/test/operators/table_scan_sorted_segment_search_test.cpp

```diff
@@ -0,0 +1,106 @@
+#include "base_test.hpp"
+#include "gtest/gtest.h"
+#include "operators/table_scan/sorted_segment_search.hpp"
+#include "storage/segment_iterate.hpp"
+
+namespace opossum {
+
+using TestData = std::tuple<std::string, PredicateCondition, int32_t, std::vector<int32_t>>;
+using Params = std::tuple<TestData, OrderByMode, bool>;
+
+namespace {
```

Comment:
anonymous namespace is usually in the top level namespace, not in namespace opossum, in Hyrise.